README.rst

```diff
@@ -28,16 +28,16 @@ If you have deployed client-side STF and want to test it locally, you can run th
 
     ansible-playbook -i `infrared workspace inventory` stf_functional_tests.yml
 
-OR
+OR::
```

Comment:
```suggestion
OR:
```

Reply:
I want the line that follows to be formatted as a literal. Only one of the colons gets into the rendered text

Reply:
I am no longer familiar with it, which is why I had to do this patch -- to fix what I thought I knew

Reply:
That's why I always just end up with markdown, I always forget all the weird syntax rules of RST.